Orange/canvas/application/outputview.py

```diff
@@ -201,119 +195,39 @@ def __exit__(self, *args):
         self.charformat = None
 
 
-class QueuedCallEvent(QEvent):
```

Comment:
IIUC it was never _needed_. If the subscribers to `TextStream.stream` signal, the only user of this class, prefer queued connections, they can mark them such in their `.connect()` calls. Otherwise, Qt signal connections are direct if the emitter and handler are in the same thread, or queued otherwise.